CHANGELOG.md

```diff
@@ -136,6 +136,7 @@ This means that druid no longer requires cairo on macOS and uses Core Graphics i
 - The `WindowHandle::get_dpi` method got replaced by `WindowHandle::get_scale`. ([#904] by [@xStrom])
 - The `WinHandler::size` method now gets a `Size` in display points. ([#904] by [@xStrom])
 - Standardized the type returned by the contexts' `text` methods. ([#996] by [@cmyr])
+- `ViewSwitcher` uses `Data` type constraint instead of `PartialEq` ([#1112] by [@justinmoon])
```

Comment:
You will also have to add a link for `[#1112]` to the others (below the names), and the dot is missing :stuck_out_tongue: 
```suggestion
- `ViewSwitcher` uses `Data` type constraint instead of `PartialEq`. ([#1112] by [@justinmoon])
```
Those aside, this looks good to me. @Zarenor what do you think?

Reply:
Fixed